src/Nethermind/Nethermind.Mev/MevPlugin.cs

```diff
@@ -158,15 +160,50 @@ public async Task<IBlockProducer> InitBlockProducer(IConsensusPlugin consensusPl
 
             _nethermindApi.BlockProducerEnvFactory = producerEnvFactory;
 
-            IManualBlockProducer standardProducer = (IManualBlockProducer)await consensusPlugin.InitBlockProducer();
-            IBeneficiaryBalanceSource standardProducerBeneficiaryBalanceSource = producerEnvFactory.LastMevBlockProcessor;
-            V1Selector v1Selector = new(BundlePool);
-            IManualBlockProducer bundleProducer = (IManualBlockProducer)await consensusPlugin.InitBlockProducer(new BundleTxSource(v1Selector, standardProducer.Timestamper));
-            IBeneficiaryBalanceSource bundleProducerBeneficiaryBalanceSource = producerEnvFactory.LastMevBlockProcessor;
-            return _nethermindApi.BlockProducer = new MevBlockProducer(new Dictionary<IManualBlockProducer, IBeneficiaryBalanceSource>()
+            if (_mevConfig.FlashbotsVersion == SelectorType.V1)
             {
-                {bundleProducer, bundleProducerBeneficiaryBalanceSource}, {standardProducer, standardProducerBeneficiaryBalanceSource}
-            });
+                IManualBlockProducer standardProducer = (IManualBlockProducer)await consensusPlugin.InitBlockProducer();
+                IBeneficiaryBalanceSource standardProducerBeneficiaryBalanceSource = producerEnvFactory.LastMevBlockProcessor;
+                V1Selector v1Selector = new(BundlePool);
```

Comment:
do not support V1, V1Selector / V2Selector was just my sketch test code in the plugin for testing efficiency of different strategies

also when using strategy, a strategy pattern should be used and not if..else

Reply:
since we are sticking with v2 only, will keep it without strategy pattern (since v2 bundle selection is the only strategy available)